### DISCUSSION TITLE
Question mark in connection url password causes "invalid port number in database URL"

### DISCUSSION THREAD

**Discussion by ColaIan** [prisma/prisma#12779] [Q&A] [answered]
I tried skipping by using \\? but it didn't work

**Comment by ColaIan** [comment, marked as answer]
Solved by URL encoding my password

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
